controllers/front/CartController.php

```diff
@@ -597,11 +597,27 @@ protected function shouldAvailabilityErrorBeRaised($product, $qtyToCheck)
      */
     protected function areProductsAvailable()
     {
+        $products = $this->context->cart->getProducts();
+
+        foreach ($products as $product) {
+            $currentProduct = new Product();
+            $currentProduct->hydrate($product);
+
+            if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === '0') {
+                return $this->trans(
+                    'The item %product% in your cart is no longer available in this version. You cannot proceed with your order until you delete or replace your product by a new one with attribute.',
```

Comment:
I think we can have a more simple/generic text and simply say "You cannot proceed with your order until you remove this product from your cart"
I'm afraid most customers won't understand the logic of "attributes"

Reply:
Or maybe we should talk about "combinations" which would be more user friendly

Reply:
It seems to be the opposite actually: talking about combinations doesn't make sense for most merchants, that's why we must introduce the notion of attributes before so that the (not-so-logic) link can be made between combinations and attributes. :-)

Here, I would like to keep the same construction as the wording about adjusting quantities but turn it into a more positive tone. So I suggest: _The item %product% in your cart is now a product with attributes. Please delete it and choose one of its combinations to proceed with your order._ in _Shop.Notifications.Error_ as well.

Reply:
Thank you @LouiseBonnard Commit done.

Reply:
@LouiseBonnard It looks like there is a conflict https://github.com/PrestaShop/PrestaShop/pull/14489/conflicts 
In fact the message is used for more generic case of available product.
What do you suggest ?

Reply:
@202-ecommerce, what do you mean by "more generic case of available product"?

Reply:
@LouiseBonnard My mistake. I've change the bad string ! New commit with the right translate had been submit